backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/snapshots/RemoveSnapshotSingleDiskLiveCommand.java

```diff
@@ -90,11 +92,6 @@ public boolean performNextOperation(int completedChildCount) {
         // This allows for recovery after a crash at any point during command execution.
 
         log.debug("Proceeding with execution of RemoveSnapshotSingleDiskLiveCommand");
```

Comment:
Just cosmetic:
Suggesting to add a new line **after**
`log.debug("Proceeding with execution of RemoveSnapshotSingleDiskLiveCommand");`
and **maybe** the preceding new lines (88 and/or 93) are now redundant.

Reply:
Made some changes

Reply:
LGTM